docs/faq.rst

```diff
@@ -360,6 +360,15 @@ Yes, by setting :setting:`DNS_RESOLVER` to ``scrapy.resolver.CachingHostnameReso
 Note that by doing so, you lose the ability to set a specific timeout for DNS requests
 (the value of the :setting:`DNS_TIMEOUT` setting is ignored).
 
+.. _faq-dataclass-items:
+
+Can I use dataclasses as items?
+-------------------------------
+
+Support for :class:`dataclasses.dataclass` objects as items was added in version 2.1.
+This works natively in Python 3.7+, or using the `dataclasses backport`_ in Python 3.6.
+See :ref:`topics-items` for more information.
+
```

Comment:
Do we need this entry, now that the documentation about items offers a section about it?